internal/auth.go

```diff
@@ -17,41 +17,41 @@ import (
 // Request Validation
 
 // Cookie = hash(secret, cookie domain, email, expires)|expires|email
-func ValidateCookie(r *http.Request, c *http.Cookie) (string, error) {
+func ValidateCookie(r *http.Request, c *http.Cookie) (string, string, error) {
```

Comment:
Rather than add another string, I'd add a struct return that contains `email` and `groups` fields, perhaps just use the `claims` struct from the server.

Reply:
(But that really depends on how far of a delta from upstream we want to go ;) )

Reply:
@jkoelker I agree